README.md

```
@@ -1,9 +1,21 @@
-# Welcome to CeyLabs!
+<p align="center">
+  <p align="center">
+   <img width="150" height="150" src="profile/assets/CeyLabs LLC.svg" alt="Logo">
+  </p>
+	<h1 align="center"><b>CeyLabs</b></h1>
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)